src/lib/__snapshots__/create-config.test.ts.snap

```diff
@@ -15,6 +15,7 @@ exports[`should create default config 1`] = `
     "styleSrc": [],
   },
   "authentication": {
+    "authDemoAllowAdminLogin": false,
```

Comment:
I know this is late, but what do you think about changing this to just `demoAllowAdminLogin`? The env name would stay the same (`AUTH_DEMO_ALLOW_ADMIN_LOGIN`), but the idea is that the `AUTH_` prefix indicates that it is part of the `authentication` object. This is the same way that `AUTH_TYPE` (which becomes `authentication.type`) works. I think that was my intention earlier, but I guess I wasn't clear enough and didn't think about it when we got this far 😅